include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -274,6 +276,13 @@ typedef struct OrtCUDAProviderOptions {
   int do_copy_in_default_stream;
 } OrtCUDAProviderOptions;
 
+typedef struct OrtOpenVINOProviderOptions {
+  const char* device_type;       // CPU_FP32, GPU_FP32, GPU_FP16, MYRIAD_FP16, VAD-M_FP16 or VAD-F_FP32
+  bool enable_vpu_fast_compile;  // Default false
```

Comment:
We've avoided the usage of bool type in C APIs so far. IIRC the reasoning for this came from [here](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/c/c_api.h#L44-L49). Are we allowing bool types now?

Reply:
Makes sense, removed. (It existed in the OpenVINO provider header, so it's gone there now too) https://github.com/microsoft/onnxruntime/blob/88c3704257e18dbee022390acbb89e4b0b7226d4/include/onnxruntime/core/providers/openvino/openvino_provider_factory.h#L9